code/testapp/src/main/java/com/adobe/marketing/optimizeapp/OffersScreen.kt

```diff
@@ -57,8 +56,8 @@ private val displayHandler: (Offer) -> Unit = { offer ->
     offer.displayed()
 }
 
-private val displayPropositionsHandler: (List<OptimizeProposition>) -> Unit = { props ->
```

Comment:
Having to make changes in this class is an indication that you have changed the public APIs of Offer class. You should ensure this runs as-is without needing any changes